plugins/module_utils/hcloud.py

```diff
@@ -94,3 +95,13 @@ def get_result(self):
         if getattr(self, self.represent) is not None:
             self.result[self.represent] = self._prepare_result()
         return self.result
+
+
+class Hcloud(AnsibleHCloud):
```

Comment:
Theoretically you could drop this shim, normally there shouldn't be someone who uses this Hcloud Base Module